### PR TITLE
Let 818 fe update the static pages footer with critical information and links

### PR DESCRIPTION
**Pull request by barbara-chaves**
This PR updates the footer according to design and adds the coming-soon link to the pages that doesn’t exist yet

UPDATE - added privacy policy page with coming soon message

## Testing instructions

Go to any static page and you should see the updated footer [design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A3594) 

## Tracking

[LET-818](https://vizzuality.atlassian.net/browse/LET-818)

### CODE REVIEW

File: frontend/layouts/static-page/footer/component.tsx [hunk 1]
```diff
@@ -216,14 +263,18 @@ export const Footer: React.FC<FooterProps> = ({
...
                   </li>
                   <li>
-                    <a className="text-gray-400 disabled:pointer-events-none">
-                      <FormattedMessage defaultMessage="FAQ’s" id="qIvPIE" />
-                    </a>
+                    <Link href={Paths.ComingSoon} passHref>
+                      <a className="text-gray-400 disabled:pointer-events-none">
+                        <FormattedMessage defaultMessage="FAQ’s" id="qIvPIE" />
+                      </a>
+                    </Link>
```
Comment (SARodrigues) [inline frontend/layouts/static-page/footer/component.tsx:270]: With #428 merged, we now have an FAQ page to link to. 
Also, the newly added links' styling doesn't match the other links in the footer (colour, underline when hovered)

File: frontend/enums/index.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export enum Paths {
   Invitation = '/invitation',
   ForInvestors = '/for-investors',
   ForProjectDevelopers = '/for-project-developers',
+  ComingSoon = '/coming-soon',
```
Comment (SARodrigues) [inline frontend/enums/index.ts:42]: I really don't think we should have a dedicated (and in the future indexed) `/coming-soon` page/url. 
See my comments [here](https://github.com/Vizzuality/heco-invest/pull/429#pullrequestreview-1053985857).

**Review by SARodrigues — CHANGES_REQUESTED** [review]
In addition to the comments, according to the acceptance criteria, `HeCo Invest Beta` should link to the homepage. 

Also (this was mentioned in the sprint preparation but I forgot to specify it in the task): could you hide the phone number and the social media title/icons?

File: frontend/enums/index.ts (hunk 2 above)
Comment (barbara-chaves) [reply frontend/enums/index.ts:42]: I did it wrong . I will fix it

**Pull request merged**